src/static/js/modules/prepare-worksheets/worksheet-config.js

```diff
@@ -223,7 +228,8 @@ this.worksheetDefaults = {
                 "alternative":"Think of all the little things that you are used to calling your landlord to deal with: a cracked window, a broken dishwasher, or a clogged toilet. As a homeowner, you will either have to fix these yourself or call and pay for a professional.",
                 "altText": "",
                 "explanation":"",
-                "deletable": false
+                "deletable": false,
+                "uneditable": true
```

Comment:
Good point. I was taking the easy route here -- if false is the most common case, you only need to specify the attribute when true. But 'editable' is probably a better/more consistent approach. I'll change it...